clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -70,17 +71,17 @@
   protected BufferedBlockOutStream mCurrentBlockOutStream;
   protected List<BufferedBlockOutStream> mPreviousBlockOutStreams;
 
-  protected final long mFileId;
+  protected final TachyonURI mPath;
```

Comment:
Sure, I've also created a larger ticket for addressing the problem at a larger scale: https://tachyon.atlassian.net/browse/TACHYON-1491